src/Exception/ApiFilterException.php

```diff
@@ -0,0 +1,11 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\ApiFilter\Exception;
+
+class ApiFilterException extends \Exception
+{
+    public static function createFrom(\Throwable $exception): self
```

Comment:
`createFrom()` is too much abstract, i'd say `createFromThrowable()` would be much better.